// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/declarationEmitObjectLiteralAccessors1.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -2,9 +2,11 @@
 
 //// [declarationEmitObjectLiteralAccessors1.d.ts]
 export declare const obj1: {
     /** my awesome getter (first in source order) */
-    x: string;
+    get x(): string;
+    /** my awesome setter (second in source order) */
+    set x(a: string);
 };
 export declare const obj2: {
     /** my awesome getter */
     get x(): string;
